### ISSUE TITLE
Getting this error SendGrid::$client

### ISSUE THREAD

**Issue by zeeshan4971** [sendgrid/sendgrid-php#248]
Hey i am getting this error 
`Cannot access protected property SendGrid::$client` when using this api to set contact is db

require 'vendor/autoload.php';
require 'lib/SendGrid.php';

$apiKey = getenv('My Api Key');
$sg = new \SendGrid($apiKey);

$url = 'https://api.sendgrid.com/'; 
$user = 'user'; // place SG username here
$pass = 'password'; // place SG password here

$request_body = json_decode('["recipient_id1", "recipient_id2"]');
$list_id = "Test_Recipts";

$response = $sg->client->contactdb()->lists()->_($list_id)->recipients()->post($request_body);
echo $response->statusCode();
echo $response->body();
echo $response->headers();

**Comment by thinkingserious** [comment]
Hello @zeeshan4971,

Thanks for reaching out to us. To help you, I will need some additional information:
1. What version of the library are you using? The latest version is 5.0.0
2. Have you looked at the example here: https://github.com/sendgrid/sendgrid-php/blob/master/examples/contactdb/contactdb.php#L117, which requires this header: https://github.com/sendgrid/sendgrid-php/blob/master/examples/contactdb/contactdb.php#L3

**Comment by zeeshan4971** [comment]
i am using the latest version

**Comment by thinkingserious** [comment]
Hello @zeeshan4971,

There are multiple issues with this code, could you tell me where you obtained it?

For example, the lines:
```
require 'lib/SendGrid.php';
$url = 'https://api.sendgrid.com/'; 
$user = 'user'; // place SG username here
$pass = 'password'; // place SG password here
```

are not needed.

Please try using the examples I posted in my previous comment. Also, you may need to update the SendGrid dependency to ensure you are using the latest version.

Thanks!

**Comment by Jabolek** [comment]
I do get the same error. Not using composer, just downloaded 5.0.0 version:

> Fatal error: Cannot access protected property SendGrid::$client

`
require("sendgrid/sendgrid-php.php");
function getSgClient(){
$sg = new SendGrid("apikey");
return $sg->client;
}
`

**Comment by thinkingserious** [comment]
@Jabolek thanks for the update!

I will look into the downloadable code to find out what is causing the error when downloaded vs using composer.

**Comment by rayfusion** [comment]
I am also getting Fatal error: Cannot access protected property SendGrid::$client using the sample code provided in this github. Any suggestions to fix?

**Comment by thinkingserious** [comment]
@rayfusion I will look into your issue also.

**Comment by rayfusion** [comment]
@thinkingserious any luck?

**Comment by thinkingserious** [comment]
Additional Bug Report:

i've stared the integration of the SendGrid WEB API in PHP from the page:

https://github.com/sendgrid/sendgrid-php

i'd like to use the 5.0.0 version, and, as the application i'm working with doesn't use Composer, i've followed the instructions from:
"Alternative: Install package from zip"

However, if i download the library using this  link:

latest packaged release of the library as a zip

It downloads the  3.2.0 version. If i try to download from the link:

⬇︎ Download Packaged Library ⬇︎

It download the 5.0.0 version, but the dependencies are not included, so the package is not ready to be used.

How can i download a 5.0.0 version of the library along with the required dependencies and use it wihtout using composer ?

**Comment by thinkingserious** [comment]
@rayfusion I'm hoping to get this resolved by end of day.

**Comment by thinkingserious** [comment]
@zeeshan4971 @rayfusion @Jabolek 

This is now fixed as of version 5.0.1 and the download links have been updated.

My apologies for the delay and thanks for your support!
